content/docs/clients/javascript/index.md

```diff
@@ -7,27 +7,20 @@ The Exceptionless JavaScript client SDK makes it easy to report errors, log deta
 
 ---
 
-If you are looking for framework-specific guides, you can jump right to them below.
+Full guides can be found below:
 
-* React
-* Vue
-* Svelte
+* [Browser (no framework)](guides/javascript-example.md)
+* [React](./guides/react-example.md)
+* [Node](./guides/node-example.md)
+* [Express](./guides/express-example.md)
 
 ---
 
-To get started, you'll need to install the Exceptionless client. There are two main ways to do this: npm and CDN.
+This quickstart focuses on the vanilla JavaScript implementation of Exceptionless. 
 
 #### npm
 
-To install with npm, run: `npm install @exceptionless/node --save`
-
-```js
-import { Exceptionless } from "@exceptionless/node";
-
-await Exceptionless.startup(c => {
-  c.apiKey: 'API_KEY_HERE'
-});
-```
+To install with npm, run: `npm install @exceptionless/browser`
```

Comment:
I guess it depends on the apps. I figured the default should use node as that's the easiest to show how to use it without extra tooling like webpack. I liked showing startup as it showed how easy it is to import and get running.

Reply:
I don't have a strong opinion here, so you and Eric decide which you'd prefer to see at the root of the JS docs and I'm happy to have it there.

Reply:
The most common use case is going to be the browser one, not the node package. So I'd agree with changing this to be the browser package.

I believe it shows `startup` being called later in this quick start. I think it's just showing the first step is to do an NPM install. I'm OK with waiting until the next step to show `startup` being used.

Reply:
Hmm... I just looked at this in whole and see that it's different sections showing how to use from NPM and then use from CDN.  I think it's probably good to show those 2 things and that both of them should live on their own. So we would want to see `startup` being called here. It's repeating information, but only a couple lines worth. I think it's worth it to have 2 functioning minimal setups one using NPM and one using CDN.

Reply:
Ok I updated this to be a lot more clear and a lot more simple. Let me know what you guys think.